projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/base/LLVMMainFunctionReturnValueRootNode.java

```diff
@@ -83,4 +83,21 @@ public Object execute(VirtualFrame frame) {
 
     }
 
+    public static class LLVMMainFunctionReturnVoidRootNode extends LLVMMainFunctionReturnValueRootNode {
+
+        // Clang uses 214 and GCC 237 for a (C) main function with void return type
+        private static final int VOID_RET_VALUE = 214;
```

Comment:
I figured out that the return value is actually whatever value is currently in `%eax`. I will update the change to return 0.